lib/__tests__/_sendEvent.test.ts

```diff
@@ -84,20 +94,189 @@ describe("sendEvent", () => {
     });
     it("should send the correct payload", () => {
       (AlgoliaInsights as any).sendEvent("click", {
-        objectID: "1"
+        eventName: "my-event",
+        indexName: "my-index",
+        objectID: "1",
+        position: 3
       });
       const payload = JSON.parse(sendBeacon.mock.calls[0][1]);
 
       expect(payload).toEqual({
         events: [
           expect.objectContaining({
             eventType: "click",
-            objectID: "1",
-            userID: "42",
+            eventName: "my-event",
+            indexName: "my-index",
+            objectID: ["1"],
+            position: [3],
+            userID: "mock-user-id",
```

Comment:
Why do we have the `userID` for the beacon but not for the XMLHttpRequest? Rather than using `objectContaining` I would use something more strict like `toEqual` since we assert the full payload.

Reply:
since the content of the payload is checked in details in other tests, i'll simplify this one and only keep required param
```js
{
    eventType: "click",
    eventName: "my-event",
    indexName: "my-index",	            
}
```